scripts/samples.json

```diff
@@ -0,0 +1,60 @@
+﻿[
+  {
+    "initializers": [
+      "nextjs"
+    ],
+    "args": {
+      "appName": "nextjs-vanilla",
+      "fetchWith": "GraphQL",
+      "prerender": "SSR",
```

Comment:
Please switch at least one of these samples to use SSG to have coverage there. Maybe also switch one to Rest?

Reply:
Also, please add "nextjs-personalize" to one of these.